releases/release-1.21/release-notes/release-notes-draft.md

```diff
@@ -142,7 +145,7 @@
 - Kubeadm: fix a bug where "kubeadm join" would not properly handle missing names for existing etcd members. ([#97372](https://github.com/kubernetes/kubernetes/pull/97372), [@ihgann](https://github.com/ihgann)) [SIG Cluster Lifecycle]
 - Kubeadm: fix a bug where "kubeadm upgrade" commands can fail if CoreDNS v1.8.0 is installed. ([#97919](https://github.com/kubernetes/kubernetes/pull/97919), [@neolit123](https://github.com/neolit123)) [SIG Cluster Lifecycle]
 - Kubeadm: fix a bug where external credentials in an existing admin.conf prevented the CA certificate to be written in the cluster-info ConfigMap. ([#98882](https://github.com/kubernetes/kubernetes/pull/98882), [@kvaps](https://github.com/kvaps)) [SIG Cluster Lifecycle]
-- Kubeadm: fix bad token placeholder text in "config print *-defaults --help" ([#98839](https://github.com/kubernetes/kubernetes/pull/98839), [@Mattias-](https://github.com/Mattias-)) [SIG Cluster Lifecycle]
+- Kubeadm: fix bad token placeholder text in `config print *-defaults --help` ([#98839](https://github.com/kubernetes/kubernetes/pull/98839), [@Mattias-](https://github.com/Mattias-))
```

Comment:
🤔 this looks like a bug in `krel` where the SIGs got overwritten

cc @puerco 

(not a blocker for this current PR though)

Reply:
same with the following:
- `98715`
- `96021` 
- `98750`

(trying to avoid unnecessarily linking the actual PRs)


...pretty much any new entries on this PR? 🤔

Reply:
Should I go and add them manually? I definitely have not touched SIG names.

Reply:
nah don't worry about it, this should be fixed in tooling -- just finish this one and we can merge: https://github.com/kubernetes/sig-release/pull/1472#discussion_r580435852

(because at that point I have `approvers` role 😄 )

Reply:
May it is something I can try to look into.

Reply:
Those entries have a map?

Reply:
I think most have maps yeah, perhaps when we apply map it's overwriting the SIGs list to be empty